docs/quickstart.md

```diff
@@ -0,0 +1,10 @@
+
+## Quickstart Guides
+
+- [Kubevirt on killercoda](https://killercoda.com/kubevirt)
+
```

Comment:
And something similar here to differentiate these quickstarts from the killacoda scenarios. My suggestion: 
```suggestion
Guides for deploying KubeVirt with different Kubernetes tools:
```